validator/src/cli.rs

```diff
@@ -433,7 +433,8 @@ pub fn port_validator(port: String) -> Result<(), String> {
 
 pub fn port_range_validator(port_range: String) -> Result<(), String> {
     if let Some((start, end)) = solana_net_utils::parse_port_range(&port_range) {
-        if end - start < MINIMUM_VALIDATOR_PORT_RANGE_WIDTH {
+        let width = end - start + 1;
```

Comment:
I think existing validation logic is correct. Ie. the port range is [start, end), note the end is not inclusive as used in various places in the code.

Reply:
Let's spell out the problem

1. User misunderstood the range usage, and gives  a -- b where b >=a, expecting to have the actual valid port counts to be b - a + 1,

Reply:
Yeah sounds reasonable.